conf/messages.en

```diff
@@ -927,4 +928,7 @@ otherExpensesDisallowableAmount.error.overAmount.individual = The disallowable a
 otherExpensesDisallowableAmount.error.overAmount.agent = The disallowable amount of the £{0} your client spent on other expenses must be less than £100,000,000,000
 otherExpensesDisallowableAmount.title.individual = How much of the £{0} you spent on other expenses is disallowable?
 otherExpensesDisallowableAmount.title.agent = How much of the £{0} your client spent on other expenses is disallowable?
+otherExpensesDisallowableAmount.change.hidden = Change how much spent on other expenses is disallowable
 
+otherExpensesCYA.title.individual = Check your other expenses
+otherExpensesCYA.title.agent = Check your client's other expenses
```

Comment:
Is this the correct apostrophe?

Reply:
i have fixed this and the messages UTs to catch this, as they currently weren't